Taskfile.yml

```diff
@@ -53,7 +53,7 @@ tasks:
     cmds:
       - docker buildx prune --all
       - docker system prune --all --volumes
-      - bin/golangci-lint cache clean
+      - bin/golangci-lint{{exeExt}} cache clean
```

Comment:
@w84thesun How did it work before for you?

Reply:
I didn't use it before 😄